srcpkgs/kdenetwork-filesharing/template

```diff
@@ -0,0 +1,20 @@
+# Template file for 'kdenetwork-filesharing'
+pkgname=kdenetwork-filesharing
+version=21.04.3
+revision=1
+build_style=cmake
+hostmakedepends="extra-cmake-modules qt5-qmake qt5-host-tools kcoreaddons
+ $(vopt_if samba 'samba') gettext ki18n kauth"
+makedepends="kconfig-devel kio-devel kdeclarative-devel
+ $(vopt_if samba 'samba-devel')"
+depends="dolphin $(vopt_if samba 'samba')"
```

Comment:
Shouldn't samba be detected automatically here?

Reply:
No. It is not added automatically to the "run" depends.